README.md

```diff
@@ -162,24 +162,45 @@ const render = () => html`
 `;
 ```
 
-### Function Values / Directives
+### Directives
 
-A function valued expression can be used for error handling and stateful rendering.
+Directives Functions are functions that can extend lit-html by directly interacting with the Part API.
 
-If an expression returns a function, the function is called with the `Part` its populating, inside a try/catch block.
+Directive Functions are created from regular functions by passing them to the `directive()` function:
 
-This makes it safe from exceptions:
+```javascript
+html`<div>${directive((part) => { part.setValue('Hello')})}</div>`
+```
+
+Directives Functions will often be returned from another function which takes arguments, which will generally be called the directive. That is, a directive is usually called inline in a template, and then returns a Directive Function to use a Part:
```

Comment:
"Directive Functions" thing again... and at the end do you want "to use _as_ a Part" ?